federated-server/shcedule/nodes.js

```diff
@@ -0,0 +1,22 @@
+import pull from 'pull-stream';
```

Comment:
s/shcedule/schedule/g  in directory name

Reply:
done